src/components/Modal/PortalChooser.js

```diff
@@ -8,6 +8,7 @@ import {
   MenuItem,
   Button
 } from '@material-ui/core';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
```

Comment:
Are we able to try use the same arrow as the "Find Place" component? Not sure if it's SVG though to allow us play with opacity.

![image](https://user-images.githubusercontent.com/877919/65427202-7e3db500-de1a-11e9-8913-b9ad25834a10.png)